projects/packages/my-jetpack/_inc/hooks/use-filtered-products/index.tsx

```diff
@@ -49,6 +51,11 @@ const useFilteredProducts = () => {
 				productsWithNoCard.push( 'stats' );
 			}
 
+			// If on multisite, filter out products that are not supported
+			if ( getScriptData().site.is_multisite ) {
+				productsWithNoCard.push( ...JETPACK_PRODUCTS_NOT_FOR_MULTISITE );
+			}
+
```

Comment:
This mostly applies to multisite installations with a plan that includes Jetpack VaultPress Backup, such as Jetpack Premium. In the future, we could consider adding a “Not supported” card status if needed.

Since we’re already displaying “Not supported on multisites” in the Products tab, hiding it from the Overview page seems like the most straightforward way to handle this for now.